contracts/tests/SpigotedLine.t.sol

```diff
@@ -753,6 +766,14 @@ contract SpigotedLineTest is Test {
       assertEq(spigot.owner(), borrower);
     }
 
+    function test_cannot_close_with_ETH() public {
+      vm.startPrank(borrower);
+        bytes32 id = line.ids(0);
+        vm.expectRevert(MutualConsent.NonZeroEthValue.selector);
+        line.close{value: 0.00001 ether}(id);
+      vm.stopPrank();
+    }
+
     function test_only_borrower_release_spigot_when_repaid() public {  
```

Comment:
IS this going to cause issues with repayment? When trading the revenue token for Eth, as `unused[Eth]` will never be used by `useAndRepay`?